core/src/main/resources/defaults/blueprints/cdp-data-engineering-702.bp

```diff
@@ -167,6 +161,10 @@
           {
             "name": "tez_container_size",
             "value": "4096"
+          },
+          {
+            "name": "hive_service_config_safety_valve",
+            "value": "<property><name>hive.server2.authentication.spnego.principal</name><value>HTTP/_HOST@CLOUDERA.SITE</value></property><property><name>hive.server2.authentication.spnego.keytab</name><value>hive.keytab</value></property><property><name>hive.hook.proto.file.per.event</name><value>true</value></property>"
```

Comment:
This hard coded value is not going to work since are not using cloudera.site anymore but it's generated on the fly. But I believe this is already handled so this config can be removed from here.

Reply:
I've to add property: <property><name>hive.hook.proto.file.per.event</name><value>true</value></property>, how do I add to the safety value without overriding the previously set one.

Reply:
Shall I change this instead? template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveKnoxConfigProvider.java

Reply:
You can add them here: 
HIVE_ON_TEZ: https://github.com/hortonworks/cloudbreak/blob/rc-2.15/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveKnoxConfigProvider.java#L25
HIVE_METASTORE: https://github.com/hortonworks/cloudbreak/blob/rc-2.15/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveMetastoreConfigProvider.java#L44

I guess you'll need it for Hive on Tez

Reply:
yup, its required for hive on tez. But since the safety_value can only be set in one config, I'll just add to the HiveKnoxConfigProvider for now, is there a way to query if its already set and append to it?

Reply:
Thanks @keyki. I've updated the code.